File: Microservices/VehicleLocationMicroservice/VehicleLocationMicroservice/Entities/VehicleData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace VehicleLocationMicroservice.Entities
{
    public class VehicleData
    {
        public int VehicleId { get; set; }
        public double VehicleSpeed { get; set; }
        public double Latitude { get; set; }
        public double Longitude { get; set; }
        public DateTime Timestamp { get; set; }

        public override string ToString()
        {
            string data = "VehicleId = " + VehicleId + "\n";
            data += "Latitude = " + Latitude + "\n";
            data += "Longitude = " + Longitude + "\n";
            data += "Timestamp = " + Timestamp.ToString() + "\n";
            return data;
        }
    }
}

File: Microservices/SparkProcessing/src/main/java/Entities/VehicleData.java
package Entities;

import com.fasterxml.jackson.annotation.JsonFormat;
import java.io.Serializable;

public class VehicleData implements Serializable{    
    @JsonFormat(shape = JsonFormat.Shape.NUMBER_INT)
    private int VehicleId;  
    @JsonFormat(shape = JsonFormat.Shape.NUMBER_FLOAT)
    private double VehicleSpeed;   
    @JsonFormat(shape = JsonFormat.Shape.NUMBER_FLOAT)
    private double Latitude;
    @JsonFormat(shape = JsonFormat.Shape.NUMBER_FLOAT)
    private double Longitude;
    @JsonFormat(shape = JsonFormat.Shape.STRING)
    private String Timestamp;
    
    public int getVehicleId() {
        return VehicleId;
    }
    
    public double getVehicleSpeed(){
        return VehicleSpeed;
    }
    
    public double getLatitude(){
        return Latitude;
    }
    
    public double getLongitude(){
        return Longitude;
    }
    
    public String getTimestamp(){
        return Timestamp;
    }
    
    @Override
    public String toString(){
        return "Vehicle [VehicleId=" + VehicleId + ", VehicleSpeed=" + VehicleSpeed + ", Latitude=" + Latitude + ", Longitude=" + Longitude
                + ", Timestamp=" + Timestamp + "]";
    }
}